lib/bolt/pal/yaml_plan.rb

```diff
@@ -6,10 +6,10 @@
 module Bolt
   class PAL
     class YamlPlan
-      PLAN_KEYS = Set['parameters', 'steps', 'return', 'version', 'description']
+      PLAN_KEYS = Set['parameters', 'private', 'steps', 'return', 'version', 'description']
```

Comment:
WIth https://github.com/puppetlabs/bolt/pull/2442 merged in, we'll want to add a definition to the schema for `private`.